cypress/integration/form.spec.js

```diff
@@ -198,4 +225,14 @@ describe('Form', () => {
           cy.wrap(res).should('have.property', 'name', 'Billy')
         })
     })
+
+    it('should navigate backwards through form', () => {
```

Comment:
Nice! I was just appreciating that I could go back in the form while adding horses.